### PR TITLE
Redshift warehouse loading tasks

### PR DESCRIPTION
**Pull request by HassanJaveed84**
We would know more about the best distribution keys/styles after we do redshift evaluation. 
It would involve evaluating the query plan to optimize the distribution style.

### CODE REVIEW

File: edx/analytics/tasks/common/redshift_load.py [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+            date=self.date,
+            **kwargs
+        )
+
+        yield LoadInternalReportingCourseCatalogToRedshift(
+            date=self.date,
+            **kwargs
+        )
+
+        yield LoadUserCourseSummaryToRedshift(
```
Comment (brianhw) [inline edx/analytics/tasks/common/redshift_load.py:493]: I assume this is still a work in progress -- I only seem to see this commented out down below.

File: edx/analytics/tasks/common/redshift_load.py [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+    @property
+    def password(self):
+        return self.credentials()['password']
+
+    @property
+    def aws_access_key_id(self):
+        return ''
+
+    @property
+    def aws_secret_access_key(self):
```
Comment (tobz) [inline edx/analytics/tasks/common/redshift_load.py:78]: Why is this (and `aws_access_key_id` above) here?  It looks like they're both unused.

**Review by tobz — APPROVED** [review]
Looks generally good to me. 👍

File: edx/analytics/tasks/common/redshift_load.py (hunk 1 above)
Comment (brianhw) [reply edx/analytics/tasks/common/redshift_load.py:493]: Nevermind, I see that it's also above.

File: edx/analytics/tasks/common/redshift_load.py [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+            **kwargs
+        )
+
+        yield DailyLoadSubjectsToRedshift(
+            date=self.date,
+            **kwargs
+        )
+
+
+# class LoadUserCourseSummaryToRedshift(RedshiftS3CopyToTable):
```
Comment (brianhw) [inline edx/analytics/tasks/common/redshift_load.py:515]: Just need to remove this -- it confused me. : )

**Review by brianhw — APPROVED** [review]
LGTM.

**Pull request closed without merging**